### PR TITLE
Minor fixes of wording and formatting

### PR DESCRIPTION
**Pull request by andrykonchin**

### CODE REVIEW

File: core/hash/to_a_spec.rb [hunk 1]
```diff
@@ -36,4 +36,14 @@
...
+
+  ruby_version_is '2.7' do
+    it "returns a not tainted array if self is tainted" do
+      {}.taint.to_a.tainted?.should be_false
+    end
+
+    it "returns a trusted array if self is untrusted" do
+      {}.untrust.to_a.untrusted?.should be_false
+    end
+  end
```
Comment (andrykonchin) [inline core/hash/to_a_spec.rb:48]: These specs complement a section above for Ruby < 2.7

File: core/io/ungetc_spec.rb [hunk 2]
```diff
@@ -103,7 +103,7 @@
     -> { @io.sysread(1) }.should raise_error(IOError)
   end
 
-  ruby_version_is "0"..."3.0" do
+  ruby_version_is ""..."3.0" do
```
Comment (andrykonchin) [inline core/io/ungetc_spec.rb:106]: Just to follow the common convention.

File: language/variables_spec.rb [hunk 3]
```diff
@@ -797,17 +797,6 @@ module VariableSpecs
...
-  # new feature in 2.6 -- https://bugs.ruby-lang.org/issues/13770
-  it 'does not allow non-ASCII upcased characters at the beginning' do
-    -> do
-      eval <<-CODE
-        def test
-          ἍBB = 1
-        end
-      CODE
-    end.should raise_error(SyntaxError, /dynamic constant assignment/)
-  end
```
Comment (andrykonchin) [inline language/variables_spec.rb:809]: The title is completely incorrect - the mentioned fix was about allowing constants to start with a non-ASCII character.

File: language/constants_spec.rb [hunk 4]
```diff
@@ -718,3 +718,17 @@ class PrivateClass
...
+      -> do
+        eval <<-CODE
+          def test
+            B = 1
+          end
+        CODE
+      end.should raise_error(SyntaxError, /dynamic constant assignment/)
+    end
+  end
+end
```
Comment (andrykonchin) [inline language/constants_spec.rb:734]: I haven't found already existing spec for this case so decided to keep it here.

File: language/variables_spec.rb (hunk 3 above)
Comment (eregon) [reply language/variables_spec.rb:809]: I think it's related to below, and that the Ruby implementation correctly notices this is upcased and so not a valid local variable name.
Could you add it back, after the spec below and change the description to something like:
> parses a non-ASCII upcased character as a constant identifier

We could also add a spec to actually set the constant in a valid context, e.g., on some module, that seems useful.

**Review by eregon — APPROVED** [review]
Looks great, thanks, I just have one comment.

File: language/variables_spec.rb (hunk 3 above)
Comment (andrykonchin) [reply language/variables_spec.rb:809]: > We could also add a spec to actually set the constant in a valid context, e.g., on some module, that seems useful.

Do you mean something like this one?

https://github.com/ruby/spec/blob/12c704ac53d4200610d48c50e0e5d1c4dd40aa82/language/constants_spec.rb#L714-L719

Comment (andrykonchin) [reply language/variables_spec.rb:809]: > Could you add it back, after the spec below and change the description to something like:

Done

Comment (eregon) [reply language/variables_spec.rb:809]: > Do you mean something like this one?

Yes, but that ones uses `const_set` which might behave differently. I think it'd be useful to have the same but with a literal constant assignment within a named module.

**Pull request merged**